app/src/main/java/com/android/sample/ui/components/AttendantPreview.kt

```diff
@@ -0,0 +1,68 @@
+package com.android.sample.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.PersonRemove
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.sample.model.image.ImageViewModel
+import com.android.sample.model.profile.User
+import com.android.sample.resources.C.Tag.LINE_STROKE
+import com.android.sample.resources.C.Tag.MEDIUM_PADDING
+import com.android.sample.resources.C.Tag.SMALL_IMAGE_SIZE
+import com.android.sample.resources.C.Tag.STANDARD_PADDING
+import com.android.sample.ui.camera.ProfileImage
+
+@Composable
+fun AttendantPreview(
+    user: User,
+    onProfileClick: (User) -> Unit,
+    imageViewModel: ImageViewModel,
+    deleteAttendant: (User) -> Unit,
+    index: Int
+) {
+  Row(
+      modifier =
+          Modifier.clickable { onProfileClick(user) }
+              .padding(MEDIUM_PADDING.dp)
+              .border(LINE_STROKE.dp, MaterialTheme.colorScheme.outline, MaterialTheme.shapes.small)
+              .testTag("attendeeRow${index}")) {
+        if (user.id == "") {
+          Icon(
+              Icons.Default.Person,
+              contentDescription = "Person",
+              modifier = Modifier.align(Alignment.CenterVertically))
```

Comment:
I think that the Icons.Default.Person should not be hardcoded in this function but in the profile logic.
If the user has no profile picture, its profile picture should always be Icons.Default.Person

Reply:
I think you maybe a little confused, using icon person default is not for profiles wihtout picture it's for participants of an activity that don't have an account (i.e. added by the cretor)

Reply:
Ok now I understan better, thank you !